snippets/ztp-example-siteconfig.adoc

```diff
@@ -80,9 +78,8 @@ spec:
 <4> Cluster labels must correspond to the `bindingRules` field in the `PolicyGenTemplate` CRs that you define. For example, `policygentemplates/common-ranGen.yaml` applies to all clusters with `common: true` set, `policygentemplates/group-du-sno-ranGen.yaml` applies to all clusters with `group-du-sno: ""` set.
 <5> Optional. The CR specifed under `KlusterletAddonConfig` is used to override the default `KlusterletAddonConfig` that is created for the cluster.
 <6> For single-node deployments, define a single host. For three-node deployments, define three hosts. For standard deployments, define three hosts with `role: master` and two or more hosts defined with `role: worker`.
-<7> Optional. Use `biosConfigRef` to configure desired firmware for the host.
-<8> Applies to all cluster types. Specifies the BMC address. ZTP supports iPXE and virtual media booting by using Redfish or IPMI protocols.
-<9> Create the `bmh-secret` CR that specifies the BMC credentials. Use the same namespace as the `SiteConfig` CR.
-<10> Use `UEFISecureBoot` to enable secure boot on the host.
-<11> Specifies the network settings for the node.
-<12> Configures the IPv6 address for the host. For {sno} clusters with static IP addresses, the node-specific API and Ingress IPs should be the same.
+<7> Applies to all cluster types. Specifies the BMC address. ZTP supports iPXE and virtual media booting by using Redfish or IPMI protocols.
+<8> Create the `bmh-secret` CR that specifies the BMC credentials. Use the same namespace as the `SiteConfig` CR.
+<9> Use `UEFISecureBoot` to enable secure boot on the host.
```

Comment:
question/suggestion on callouts <8> and <9>. Is callout <8> telling the user to create the `bmh-secret` CR prior to installation, or is it saying that setting this variable in the config file will create the `bmh-secret` for them? If the CR is created for them, I think this should be `Creates the ...` and `Uses the ...` to match tense.

Same with callout <9>, it could be reworded like `Specifies whether secure boot is enabled on the host` to match tense.

Reply:
Well spotted thanks. Updated to clarify.